backend/src/LibBinarySerialization/LibBinarySerialization.fsproj

```diff
@@ -19,8 +19,7 @@
     <None Include="paket.references" />
     <Compile Include="SerializedTypes.fs" />
     <!-- TODO: consider merging the below 2 files into one module? -->
-    <Compile Include="SerializedTypesToProgramTypes.fs" />
-    <Compile Include="ProgramTypesToSerializedTypes.fs" />
+    <Compile Include="ProgramTypesSerializedTypes.fs" />
```

Comment:
Probably best to rename this to match the other style - perhaps ProgramTypesToSerializedTypes?